CMakeLists.txt

```diff
@@ -246,7 +246,11 @@ endif()
 if(ARB_WITH_PYTHON)
     cmake_dependent_option(ARB_USE_BUNDLED_PYBIND11 "Use bundled pybind11" ON "ARB_WITH_PYTHON;ARB_USE_BUNDLED_LIBS" OFF)
 
-    find_package(Python3 ${arb_py_version} COMPONENTS Interpreter Development REQUIRED)
+    if(DEFINED ENV{CIBUILDWHEEL})
+        find_package(Python3 ${arb_py_version} COMPONENTS Interpreter Development.Module REQUIRED)
```

Comment:
This conditional seems weird, I would expect that we can always load either Development.Module, Development, or both and have success?

Reply:
Unfortunately, local builds with `Development.Module` fail, and in-container builds fail with `Development` (this seems to be on purpose). The only piece of documentation I could find is what I linked, but I don't understand it, only that it works.